actions/pyproject.toml

```diff
@@ -13,16 +13,16 @@ python = "^3.10"
 cryptography = "^43.0.1"
```

Comment:
It seems `cryptography` also has an update, so, it could be:

`>=43.0.1,<46.0.0